discovery-infra/test_infra/cluster.py

```diff
@@ -0,0 +1,237 @@
+import logging
+
+from tests.conftest import env_variables
+from test_infra import consts, utils
+
+
+class Cluster:
```

Comment:
maybe lets add folder for such sort of objects? not sure what its name but will be nice to have it in folder.
help_classes? maybe